job-dsl-plugin/src/main/groovy/javaposse/jobdsl/plugin/JobDslPlugin.groovy

```diff
@@ -0,0 +1,41 @@
+package javaposse.jobdsl.plugin
+
+import hudson.Plugin
+import jenkins.model.Jenkins
+import net.sf.json.JSONObject
+import org.kohsuke.stapler.StaplerRequest
+import org.kohsuke.stapler.StaplerResponse
+
+import javax.servlet.ServletException
+
+import static hudson.model.UpdateCenter.ID_DEFAULT
+
+class JobDslPlugin extends Plugin {
+    private JSONObject api
+    private String version
+
+    @Override
+    void postInitialize() throws Exception {
+        version = wrapper.version.split()[0]
+
+        File baseDir = new File(wrapper.baseResourceURL.toURI())
+        api = JSONObject.fromObject(new File(baseDir, "api-viewer/build/data/dsl-${version}.json").text)
+        api.element('embedded', true)
+    }
+
+    @Override
+    void doDynamic(StaplerRequest request, StaplerResponse response) throws IOException, ServletException {
+        String path = request.restOfPath
+        if (path == '/api-viewer') {
+            response.sendRedirect("${request.requestURI}/index.html")
+        } else if (path == '/api-viewer/build/data/update-center.jsonp') {
+            JSONObject data = Jenkins.instance.updateCenter.getById(ID_DEFAULT).JSONObject
```

Comment:
I had to add 
```groovy
response.setContentType('application/javascript')
```
here to get this to work. I was getting:
```
Refused to execute script from 'http://localhost:8080/plugin/job-dsl/api-viewer/build/data/update-center.jsonp' because its MIME type ('') is not executable, and strict MIME type checking is enabled.
```